helper/versions/version.go

```diff
@@ -52,3 +53,19 @@ func GetBuiltinVersion(pluginType consts.PluginType, pluginName string) string {
 
 	return DefaultBuiltinVersion
 }
+
+func IsBuiltinVersion(v string) bool {
+	semanticVersion, err := semver.NewSemver(v)
```

Comment:
nit: we could avoid extra version parsing by returning `false` when `v == ""`

Reply:
If you follow through the call-path for the case when `v == ""`, it's really very lightweight, and this code isn't called in any hot paths, so I'm going to leave this one as premature optimisation, because I don't think it improves readability.